src/assessment.js

```diff
@@ -2,52 +2,61 @@
 
 // Study the following code. Then answer the questions that follow.
 //
+// NOTE: You must answer the first four questions before answer the last
+// question. The last question asks you to change this code in a way that
+// renders previous answers nonsensical.
+//
 // NOTE: Line numbers are annotated line numbers, not editor line numbers.
 
-/*1*/ var alderaan = "Boom";
-/*2*/ var deathStar = function deathStar(){
-/*3*/   var laserQuantity = 1;
-/*4*/   var laserSize = "huge";
-/*5*/   var fire = function fire(){
-/*6*/     // some code
-/*7*/   };
+/*1*/ var alderaan = { name: 'Alderaan' };
+/*2*/ var deathStar = {
+/*3*/   laserQuantity: 1,
+/*4*/   laserSize: "huge",
+/*5*/   fire: function fire(){
+/*6*/     conosole.log(this);
+/*7*/   }
 /*8*/ };
 
 // Suppose we had declared another function between line 7 and 8. Would that
-// new function have access to the value stored in the variable `laseQuantity`
+// new function have access to the value stored in the variable `laserQuantity`
 // on line 3?
 //
 //     `true` or `false`
 //
-var q1 = /* your answer here */;
+var q1 = true; //the new function would still be in the scope of deathstar
 //
 
-// Which scope(s) may access the variable `laswerQuantity`? Choose one or many
+// Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
+var q2 = ['deathstar', 'fire'];
 //
 
-// Study the following code. Then answer the questions that follow.
+// Which of the following are defined within the scope of the `fire` function?
+// Choose one or all of the following:
+//
+//     `['alderaan', 'chartreuse', 'frickinLasers', 'laserSize']`
+//
+var q3 = ['alderaan', 'laserQuantity', 'laserSize'];
```

Comment:
Although `laserQuantity` wasn't on the list, you're correct in that it is accessible from `fire`.